docker-compose.hub.core.yaml

```
@@ -1,103 +1,93 @@
-    env_file:
-      - env/event-flow.env
-      - .env
-    environment:
-      - CONFIGURATION_DATABASE_USER=postgres
-      - CONFIGURATION_DATABASE_PASSWORD=unused
-    restart: always
-    depends_on:
-      - postgres
-      - valkey
-      - nats
+services:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)